frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -251,6 +251,15 @@ If you'd like to change the order in which dictionaries are queried (and their r
                     })
                 end,
             },
+            { -- setting used by dictquicklookup
+                text = _("Full screen"),
```

Comment:
What do you think about the `Large window` suggestion?

Reply:
It sounds good to me.